databrowser/src/components/select/SelectCustom.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <select
+    v-model="value"
+    class="
+      py-1
+      pr-6
+      pl-4
+      rounded-full
+      focus:border-green-500
+      appearance-none
+      outline-none
+      bg-white bg-no-repeat
+      border border-gray-500
+    "
+  >
+    <slot></slot>
+  </select>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/runtime-core';
+
+export default defineComponent({
+  props: {
+    modelValue: {
```

Comment:
Could we just name it `value`?

Reply:
If I'm not wrong this is the name-by-convention to attach a `v-model` attribute, it won't work with another property name

Reply:
thanks for the explanation. 👍